src/ert/scheduler/lsf_driver.py

```diff
@@ -325,8 +325,13 @@ async def poll(self) -> None:
                     f"bjobs gave returncode {process.returncode} and error {stderr.decode()}"
                 )
             bjobs_states = _Stat(**parse_bjobs(stdout.decode(errors="ignore")))
-
-            if missing_in_bjobs_output := set(current_jobids) - set(
+            current_timestamp = int(time.time())
+            expected_job_ids = {
```

Comment:
I don't get what "expected" means here. What about `not_recent_job_ids`, `aged_job_ids` or something along that?

Reply:
By "expected", I mean that those are the job_ids we expect to be found in bjobs, so when we do `expected_job_ids - found_job_ids`, we would get the missing ones.  
I am not sure if "not_recent_job_ids" would be a good name, as we might add more filtering down the line (pagination or something like that), and "recent" is very abstract. 
What about "job_ids_expected_found_in_bjobs_output"?

Reply:
Yes, go for the last one